__tests__/app.test.js

```diff
@@ -65,6 +65,7 @@ describe('/api/articles/:article_id', () => {
                     created_at: '2020-08-03T13:14:00.000Z',
                     article_img_url:
                       "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                    comment_count: "2"
```

Comment:
This might make more sense as a number than a string.

Also, there is nothing wrong with a new test for this. Depending how you wrote the first test it may have been quite brittle that this test failed now you've added the key. In an ideal world you wouldn't change any previous tests.